scripts/push-vellum-workflows.ts

```diff
@@ -392,6 +396,71 @@ class VellumWorkflowPusher {
     }
   }
 
+  private async handleServicesChanges(): Promise<void> {
+    try {
+      const gitStatus = execSync("git status --porcelain vellum/services/", {
+        encoding: "utf8",
+        cwd: process.cwd(),
+      }).trim();
```

Comment:
git status won't work bc we will have a clean working dir. we will need to do some sort of git diff command between the current branch and origin main